framework/src/Volo.Abp.EntityFrameworkCore/Volo/Abp/Domain/Repositories/EntityFrameworkCore/IEfCoreBulkOperationProvider.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Volo.Abp.Domain.Entities;
+using Volo.Abp.EntityFrameworkCore;
+
+namespace Volo.Abp.Domain.Repositories.EntityFrameworkCore
+{
+    public interface IEfCoreBulkOperationProvider
```

Comment:
Should we consider the entity with the custom id type?

```cs
Task InsertManyAsync<TDbContext, TEntity, TKey>(
        IEfCoreRepository<TEntity, TKey> repository,
        IEnumerable<TEntity> entities,
        bool autoSave,
        CancellationToken cancellationToken
    )
```

Reply:
That might be useful